region/maridia/inner-pink.json

```diff
@@ -1317,86 +830,10 @@
                       {"canShineCharge": {
                         "usedTiles": 22
                       }},
-                      "canPassBombPassages",
-                      "canUseSpringBall",
-                      "can2HighWallMidAirMorph",
-                      {"and": [
-                        "canSuitlessMaridia",
-                        "Morph"
-                      ]}
-                    ]}
-                  ]},
-                  {"and": [
-                    "canSuitlessMaridia",
-                    "Morph",
-                    {"or": [
-                      "canSpringBallJumpMidAir",
-                      "HiJump",
-                      "Ice"
-                    ]}
-                  ]}
-                ]}
-              ],
-              "unlock": null
-            },
-            {
-              "id": 2,
-              "requires": [
-                {"or": [
-                  {"and": [
-                    "canSwim",
-                    {"or": [
-                      {"canShineCharge": {
-                        "usedTiles": 22
-                      }},
-                      "canPassBombPassages",
-                      "canUseSpringBall",
-                      "can2HighWallMidAirMorph",
-                      {"and": [
-                        "canSuitlessMaridia",
-                        "Morph"
-                      ]}
-                    ]}
-                  ]},
-                  {"and": [
-                    "canSuitlessMaridia",
-                    "Morph",
-                    {"or": [
-                      "canSpringBallJumpMidAir",
-                      "HiJump",
-                      "Ice"
-                    ]}
-                  ]}
-                ]}
-              ],
-              "unlock": null
-            },
-            {
-              "id": 3,
-              "requires": [
-                {"or": [
-                  {"and": [
-                    "canSwim",
-                    {"or": [
-                      {"canShineCharge": {
-                        "usedTiles": 22
-                      }},
-                      "canPassBombPassages",
-                      "canUseSpringBall",
-                      "can2HighWallMidAirMorph",
-                      {"and": [
-                        "canSuitlessMaridia",
-                        "Morph"
-                      ]}
-                    ]}
-                  ]},
-                  {"and": [
-                    "canSuitlessMaridia",
-                    "Morph",
-                    {"or": [
-                      "canSpringBallJumpMidAir",
-                      "HiJump",
-                      "Ice"
+                      {"canComeInCharged": {
+                        "fromNode": 4,
+                        "framesRemaining": 180
```

Comment:
Tested this out and got it done in ~100 frames so 120 should be a good fit.

Reply:
Oh yeah, you can shinespark. I was only thinking of achieving blue suit by the end of the runway and not sparking at all. I'll add this as an alternate strat though.